src/training/_hooks.py

```diff
@@ -43,9 +43,9 @@ def after_run(self, context, values):
 
 
 class LogSessionHook(tf.train.SessionRunHook):
-  """Logs the session loop by outputting steps, and throughput into logs.
+  """Writes training throughput stats into the logs.
   """
-  _MESSAGE_FORMAT = 'Run: %.2f sec; Steps: %d; Duration: %d sec; Throughput: %.1f instances/sec'
+  _MESSAGE_FORMAT = 'Training throughput: Step %d time: %.2f; Duration: %d sec; Lifetime throughput: %.1f instances/sec'
```

Comment:
I don't think run_time is the time between log events. Before_run and after_run get called on each training step, right? So in after_run(), run_time = time of running 1 training step, not N steps.

That's why I was confused by which values cover just the last step (run, loss), the last N steps between successive logs (nothing?), and all steps (duration, throughput)